watchpoints.c

```diff
@@ -49,14 +49,15 @@ struct perf_event *watchpoints[WATCHPOINTS_MAX];
 struct change_list changes;
 
 
-static void watchpoint_handler(struct perf_event *bp, struct perf_sample_data
-			       *data, struct pt_regs *regs);
-static long watchpoints_ioctl(struct file *file, unsigned int cmd, long
-			      unsigned ptr_message);
+static void watchpoint_handler(struct perf_event *bp, 
+			       struct perf_sample_data *data, 
+			       struct pt_regs *regs);
+static long watchpoints_ioctl(struct file *file, unsigned int cmd, 
+			      long unsigned ptr_message);
```

Comment:
When aligning arguments to the function, please align under the first argument and not the opening parenthesis